round4/panchel.md

```diff
@@ -0,0 +1,95 @@
+# Chrome Extension Idea: ProfitPeak
+
+## Authors
+
+panchel, rashaik, vazmarc, abdellah
+Ahmed Abdellah, Chelsea Pan, Rayyan Shaik, Marc Vaz
+
+## Problem Statement
+
+Managing cryptocurrency trades across multiple wallets, chains, and platforms can be a daunting task, especially for active traders and portfolio managers. Manually tracking buy and sell transactions, calculating profit and loss (PnL), and generating comprehensive reports can be time-consuming and error-prone. Additionally, setting up stop-losses and managing risk across various positions can be challenging, particularly when trading opportunities arise at any time of the day or night.
+
+## Target Audience
+
+The target audience for ProfitPeak includes active crypto traders and institutional portfolio managers. Active traders are individuals who frequently buy and sell tokens across various platforms and wallets, seeking tools for automatic trade detection to capitalize on market movements. On the other hand, portfolio managers manage diversified assets across multiple wallets and networks. They need features like aggregated performance reporting, multi-wallet and cross-chain support, and risk-management tools to optimize their strategies. These managers often must report performance to senior level executives and generating performance data visualizations are typically a hazard. 
+
+## Description
+
+The primary goal of the ProfitPeak Chrome extension is to provide a streamlined solution for cryptocurrency traders and portfolio managers to effortlessly monitor and manage their trades, positions, and portfolio performance across multiple wallets and blockchain networks. The extension aims to achieve the following objectives:Automatic Trade Detection, PnL Calculation and Reporting, Risk Management, Cross-Chain and Multi-Wallet Support, and a Customizable User Experience
+
+## Selling Points
+
+1. Time-Saving: By automating trade detection and PnL calculations, users can save significant time and effort that would otherwise be spent manually tracking and consolidating data from multiple sources.
+2. Accuracy and Consistency: With automated trade recognition and calculations, the extension eliminates the risk of human error and ensures consistent and accurate reporting of portfolio performance.
```

Comment:
Doesn't this feature already exist on all investing apps? (average cost/overall pnl)

Is the pull of the app that these existing features are all integrated into one space? Or are there novel portfolio performance tracking features that the team is planning to develop?

If it's the latter, I've personally found that measuring a stock's performance based on the average cost of my purchases is a bit uninformative. For example, if I buy 1 BTC at these respective prices 50k, 60k, 70k, there's no way to gauge the performance of each of your buys and sells without manually doing these calculations. I feel like having these would help me better learn which strategies do and don't work.
 
I hope that makes sense, and feel free to follow up with any questions!